app/controllers/stars_controller.rb

```diff
@@ -1,2 +1,36 @@
 class StarsController < ApplicationController
+
+  def new
+    @star = Star.new
+  end
+
+  def create
+    @star = Star.new(star_params)
+    if @star.save
+      redirect_to star_path(@star)
+    else
+      render :new
+    end
+  end
+
+  def edit
+    set_star
+  end
+
+  def update
+    set_star
```

Comment:
could be moved to the top of the Class
`before_action :set_star, only: [:edit, :update]`

Reply:
thibault l'a fait dans son code je crois donc on résoudra ça à la fin il y aura plusieurs conflits

Reply:
ok